packages/cli/src/dbt/targets/bigquery.ts

```diff
@@ -0,0 +1,116 @@
+import { JSONSchemaType } from 'ajv';
```

Comment:
Should we remove this from this PR or do we want to support bigquery in this PR ?

Reply:
deleted!